### ISSUE TITLE
$price attribute provided by Steam Wishlist integration not displaying

### ISSUE THREAD

**Issue by Andrewpaps** [NemesisRE/upcoming-media-card#2]
## Describe the bug:
'$price' attribute correctly provided by Steam Wishlist integration incorrectly being recognized/displayed.

## Config YAML:
`type: custom:upcoming-media-card`
`entity: <sensor.steam_wishlist_123etc>`
`image_style: fanart`
`collapse: price=🎫`

## Components:
1. Steam Wishlist custom component (Latest , ver. v2.4.0)
2. Upcoming Media Card custom component (Latest, ver. 0.4.4)

## HA Error:
No log error.

## Troubleshooting Guide:
Guide read & followed.

## Home Assistant Version:
Latest (2024.6.2)

## Screenshots:
![image](https://github.com/NemesisRE/upcoming-media-card/assets/71562383/7247849a-cd40-4052-a288-b7f2c759d7b6)

## Additional Context:
Upcoming Media Card is used as an example on the Steam Wishlist repo, so it seems as if this did work at some stage previous.

**Comment by mkanet** [comment]
@Andrewpaps thats because you're not using the [original UMC repo](https://github.com/custom-cards/upcoming-media-card ).

You will need to temporarily add this repo to your HACS instalation. Until the original repo is officially added back to HACS.

**Comment by Andrewpaps** [comment]
Yep, I had just figured out this was the issue! Thanks for the confirmation.
